src/components/form/radio/radio.test.tsx

```diff
@@ -64,5 +64,13 @@ describe('EuiRadio', () => {
 
       expect(component).toMatchSnapshot();
     });
+
+    test('labelProps is rendered', () => {
+      const component = render(
+        <EuiRadio id="id" onChange={() => {}} labelProps={requiredProps} />
```

Comment:
Same problem here where in order for the snapshot to render the `labelProps` you need to also pass a label
```suggestion
        <EuiRadio id="id" onChange={() => {}} label="Label" labelProps={requiredProps} />
```